conduct.md

```diff
@@ -11,8 +11,49 @@
 * Attendance
 * Deadlines
 * Communication
-* Participation
-* Decisions
+
+### Participation
+
+Detailed paragraph describing the category. (delete when added paragraph)
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * points
+
+* **Unacceptable Conduct (U):**
+  * *Members shall not*
+    * points
+
+### Decisions
+
+A 3/4th majority of people in a group should agree upon a given decision before it's
```

Comment:
Shouldn't it just be either 3/4, three fourths, or majority?

Reply:
@enpuyou I agree, I don't think it needs to explicitly be 3/4ths as there won't always be 4 people in a group.

We should just change it the to majority of the group.

Reply:
@enpuyou I have made this change.